<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_widget</class>
 <widget class="QWidget" name="main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vectores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc_files/resources.qrc">
    <normaloff>:/icon/Images/math.ico</normaloff>:/icon/Images/math.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#main_widget{
background-color: #d3d3d3;

}
QTabWidget:pane{
background-color: #d3d3d3
}
#vector_x_vector_widget{
background-color: #d3d3d3;
}
QTabBar::tab{
background-color: #80857C;
color: white;
}

QTabBar::tab:hover{
background-color: #B0B3AD
}

QTabBar::tab:selected{
background-color:#454742
}

QSpinBox{
color: white;
background-color: #6c757d;
}
QSpinBox:up-button{
image:url(:/icon/Images/arrow_upwards.png);
border-left:0.5px solid #fff;
subcontrol-position:right;
width: 20px;
height: 20px;
}
QSpinBox:down-button{
image:url(:/icon/Images/arrow_downwards.png);
border-right:0.5px solid #fff;
subcontrol-position:left;
width:20px;
height:20px
}
QSpinBox:up-button:hover{
background-color: #5c636f;
}
QSpinBox:down-button:hover{
background-color: #5c636f;
}
QPushButton{
border: 2px solid #404040;
border-radius: 8px;
padding: 1px 5px;
color: #fff;
background-color: #6c757d;
font: 700 9pt &quot;Calibri&quot;;
}

QPushButton:hover{
background-color: #5c636a;
}
QPushButton:pressed {
    border: 4px solid #c0c4c8 
}
QLabel {
    color: #fff;
 background-color: #6a858d; 
    border:2px solid #404040;        
    border-radius: 4px;           
    padding: 2.5px 5px;            
font: 700 9pt &quot;Calibri&quot;;
}

QTableWidget{
background-color: #819299; 
gridline-color: #fff;  
}

QHeaderView{
background-color: #85939a;
color:white;
font: 700 9pt &quot;Calibri&quot;;
}

QHeaderView::section {
background-color: #85939a;
}
QTableCornerButton::section {
background-color: #85939a;
}
QTableWidget::item{
color:#fff
}
QTableWidget::item::hover{
background-color: #B6BEC2
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="vector_x_vector_widget">
      <attribute name="title">
       <string>Vector por vector</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="vxv_options_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="vxv_matrix_vector_widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,100">
             <item>
              <widget class="QWidget" name="vxv_spinbox_widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <property name="topMargin">
                 <number>9</number>
                </property>
                <property name="rightMargin">
                 <number>9</number>
                </property>
                <property name="bottomMargin">
                 <number>9</number>
                </property>
                <item alignment="Qt::AlignmentFlag::AlignHCenter">
                 <widget class="QLabel" name="vxv_row_label">
                  <property name="text">
                   <string>Fila de la columna</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="vxv_row_spinbox">
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="vxv_row_vector_widget" native="true">
               <property name="styleSheet">
                <string notr="true">QTabBar:tab{
max-width:10px;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item alignment="Qt::AlignmentFlag::AlignTop">
                 <widget class="QTableWidget" name="vxv_row_vector">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QTableWidget:item{
max-width:10px;
}</string>
                  </property>
                  <property name="gridStyle">
                   <enum>Qt::PenStyle::SolidLine</enum>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="horizontalHeaderMinimumSectionSize">
                   <number>10</number>
                  </attribute>
                  <attribute name="horizontalHeaderDefaultSectionSize">
                   <number>50</number>
                  </attribute>
                  <attribute name="horizontalHeaderHighlightSections">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderDefaultSectionSize">
                   <number>22</number>
                  </attribute>
                  <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <row/>
                  <column>
                   <property name="text">
                    <string/>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string/>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string/>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string/>
                   </property>
                  </column>
                  <item row="0" column="0">
                   <property name="text">
                    <string/>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="vxv_column_vector_widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="vxv_column_vector">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>10</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>35</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>35</number>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>35</number>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <row>
                <property name="text">
                 <string>Nueva fila</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Nueva fila</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Nueva fila</string>
                </property>
               </row>
               <row>
                <property name="text">
                 <string>Nueva fila</string>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>b</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="vxv_solution_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,100">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="vxv_solution_options" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignmentFlag::AlignLeft">
              <widget class="QPushButton" name="vxv_solve_scalar_button">
               <property name="text">
                <string>Obtener escalar</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignmentFlag::AlignLeft">
              <widget class="QLabel" name="vxv_scalar_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Valor</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="vxv_horizontal_spacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="matrix_x_matrix_widget">
      <attribute name="title">
       <string>Matriz por matriz</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="vector_x_matrix_widget">
      <property name="accessibleName">
       <string>asd</string>
      </property>
      <attribute name="title">
       <string>Vector por matriz</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qrc_files/resources.qrc"/>
 </resources>
 <connections/>
</ui>
